administrator/components/com_joomgallery/sql/updates/mysql/3.7.1.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `#__joomgallery_orphans` CHANGE `fullpath` `fullpath` varchar(500) NOT NULL;
+ALTER TABLE `#__joomgallery_orphans` DROP INDEX `fullpath`, ADD INDEX `fullpath` (`fullpath`(500)) USING BTREE;
```

Comment:
`USING BTREE` is not needed as `BTREE` in anyway the default indexing algorythm.

Reply:
Changed.